data/human/human missions.txt

```diff
@@ -6849,3 +6849,318 @@ mission "FW Refugees to Humanika"
+
+			label showtime
+			`	"Showtime!" he says, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, land near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
+			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky as if he sees something. He motions, and the Quicksilvers fire up their engines.`
+			`	"Oh, I don't like the look of this," says Bunker.`
+			choice
+				`	"Keep filming."`
+				`	"Call back the drones immediately."`
+					goto "call drones"
+				`	"I think we'll have to leave the drones behind."`
+					goto "no can do"
+				`	"Did he see us?"`
+					goto "see us"
+
+			`	"I don't know. I think we oughta' get going," says Bunker, grimacing.`
+			choice
+				`	"That's an order!"`
+				`	"Okay. Call the drones back."`
+					goto "call drones"
+
+			`	"You're the captain," Bunker responds in a sore tone. Meanwhile, you continue to watch the display. It takes a couple minutes before the last of the slaves are loaded, followed by the security team. The two men in the suit and hood are the last to board their ships.`
+			`	You notice a slight hum from outside before getting raked with particle fire from the pirate Quicksilvers. You try to pull out to space but another pair of Quicksilvers appear, sending even more volleys of particle fire at you. The <ship>'s repulsors are hit and you feel your pilot seat go out underneath you as your ship falls from the sky, exploding on impact with <origin>'s surface.`
+				die
+
+			label "see us"
+			`	"I have no idea," yells Bunker. "Right now we have bigger things to worry about!"`
+			choice
+				`	"Call back the drones immediately."`
+					goto "call drones"
+				`	"I think we have to leave the drones behind."`
+
+			label "no can do"
+			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones we lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`
+			choice
+				`	"We're leaving. Now!"`
+				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
+					goto escape
+
+			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
+			choice
+				`	(Charge him with mutiny.)`
```

Comment:
Thinking on it, this guy isn't part of your crew. Also, you as a captain don't have legal rights to have someone charged.

Reply:
I think under maritime law the captain of a ship in international waters can charge somebody. Since this ship is on a planet, though, I’d imagine local law would technically need to be involved. As for whether it is actually mutiny if the person isn’t technically part of your crew? No idea. Would it be attempted hijacking, maybe?

Reply:
I think it doesn't matter if he's a member of your crew. He's required to obey the captain while aboard your vessel.

Reply:
```suggestion
				`	(Force him to stand down.)`
```
It's not exactly clear as-is that picking this option results in `first last` pulling their gun. I think this is a little more clear and resolves the question of mutiny vs. hijacking, but open to suggestions.